couchdb/tests/client.py

```diff
@@ -374,6 +374,76 @@ def test_query_multi_get(self):
         for idx, i in enumerate(range(1, 6, 2)):
             self.assertEqual(i, res[idx].key)
 
+    def test_find(self):
+        if self.server.major() < 2:
```

Comment:
I tried to use the `skipTest` function, but python 2.6 does not have this, see [here](https://docs.python.org/2.6/library/unittest.html).